_pages/speaking-youtube-checklists.md

```diff
@@ -19,7 +19,7 @@ title: Speaking Checklists for YouTube Videos
 
 {% capture youtube-copy-title %}copy-{{ post.slug | slugify }}-youtube{% endcapture %}
 
-<h4>{{ post.date | date: "%b %d %l:%M %p %Z" }} - <div id="{{ youtube-copy-title }}">{{ post.title }}</div></h4>
+<h4><!--{{ post.date | date: "%b %d %l:%M %p %Z" }} - -->DjangoCon US 2022 - <div id="{{ youtube-copy-title }}">{{ post.title }}</div></h4>
```

Comment:
Slight tweak so that the copy picks inside the div and puts DCUS at the end. This is slightly different than previous years but it's less annoying IMO. 

```suggestion
<h4><!--{{ post.date | date: "%b %d %l:%M %p %Z" }} - --><div id="{{ youtube-copy-title }}">{{ post.title }} - DjangoCon US 2022</div></h4>
```